Simulator/Test/test_conversions.py

```diff
@@ -76,7 +77,57 @@ def test_quatrot():
     quat = np.array([1, 0, 1, 0])
     ans = np.array([0, 1, -1])
     np.testing.assert_allclose(ans, conv.quatrot(quat/np.linalg.norm(quat), vec), atol=tol)
-    
+
+def test_L():
```

Comment:
Testing `L` and `R` is nice, but maybe redundant. R and L are just means to an end (the quarternion rotation in `quatrot`). In fact, even if we ignore that the current implementation of `L` and `R` is silly and pretend it was ideal, it would still probably be better to hard-code the rotation in `quatrot` to avoid allocation. So anyway all of this rambling was to say that these tests might not be necessary.

Reply:
I can remove the tests, but someone also needs to change the actual functions

Reply:
Also, L and/or R can be used for quaternion multiplication (a potential application other than rotating a vector) so it makes sense to keep at least one of them. But I agree the rotation could be hard coded into quatrot.

Reply:
Fair enough, I guess there's no harm in keeping them